### PR TITLE
add my fact

### PR DESCRIPTION
**Pull request by SrunNalin**

### CODE REVIEW

**Comment by SrunNalin** [comment]
@Javanochka check please

File: class01-git/facts.py [hunk 1]
```diff
@@ -75,5 +77,5 @@ def load_answers():
         print('> ', end='', flush=True)
         try:
             run_cmd(sys.stdin.readline().strip())
-        except Exception: # This is a VERY bad style. 
+        except Exception: # This is a VERY bad style.
```
Comment (Javanochka) [inline class01-git/facts.py:80]: This change was unnecessary!

File: class01-git/facts.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
            facts=['Fact-right.', 'Fact-write.', 'Fact-right.']),
     Person(github_login='javanochka', first_name='Anna', last_name='Nikiforovskaja',
            facts=['I can meow like a cat', 'I am a PhD student', 'I am not Estonian']),
+    Person(github_login='srunnalin', first_name='Nalin', last_name='Srun',
+           facts=['I like to sleep in my freetime :)', 'I am a master student', 'I am from Cambodia']),
```
Comment (Javanochka) [inline class01-git/facts.py:13]: Which fact is the wrong one?

**Review by Javanochka — CHANGES_REQUESTED** [review]
Please, remove the unnecessary change and tell which fact is wrong (if you're ok with that)

**Review by Javanochka — APPROVED** [review]

**Pull request merged**